lib/commonAPI/mediacapture/ext/platform/android/src/com/rho/camera/CameraObject.java

```diff
@@ -139,8 +147,8 @@ public void onPictureTaken(byte[] data, Camera camera) {
             }
             bitmap.recycle();
             mPreviewActivity.finish();
-            
-        } 
+            mcameraActivity.finish();
```

Comment:
This is unneeded. mcameraActivity is same object as mPreviewActivity. You call finish() twice.